src/content/blog/my-experience-at-jsheroes-2023.md

```diff
@@ -0,0 +1,34 @@
+---
+author: alex-pavaloi
+author_category: guest-writers
+title: My Experience at JSHeroes 2023
+published: Dec 8, 2023
+tags: ["general"]
+summary: Hi! I'm Alex and this blog post is about my experience at JS Heroes 2023.
+---
+
+After a couple of years without in-person conferences, I was really glad to return to JSHeroes for a 2-day event. So I'm gonna use this blog post to share my experience with you.
+
+I have to start with the MC - Anjana Vakil - which I think was a perfect fit for the role. She was so full of energy and enthusiasm that she really got us pumped for the conference. I wish I could embed some video snippets here so you could see her contagious energy for yourself.
```

Comment:
You can link the after movie maybe? (https://www.youtube.com/watch?v=N6qr6L37sl8) she's quite visible at the beginning :) youtube link can also be embedded I think

Reply:
good idea. Linked the movie, found a part at the beginning that really fits what the paragraph is about.